force-app/main/wageClaim/classes/HOT_WageClaimListController.cls

```diff
@@ -0,0 +1,36 @@
+public without sharing class HOT_WageClaimListController {
```

Comment:
Kunne vi satt opp et sharingSet for frilanstolkene så vi ikke trenger å bruke without sharing?

Reply:
Ja

Reply:
`This list includes only objects with an organization-wide default of Private or Public Read Only. These objects must have an account or contact lookup field, or point to another object which does`

Siden WageClaim er controlled By Parent, vil de vel ha tilgang uansett... Prøver